Dockerfile

```diff
@@ -1,11 +1,12 @@
 # DOCKER-VERSION 0.4.0
 
-FROM ubuntu:trusty
+FROM ubuntu:bionic
```

Comment:
nit: `focal` would be available as latest LTS if we wanted the latest(best?) versions of the involved packages

Reply:
I just tested a `docker build` with `ubuntu:focal` and the build failed with this error, so I would keep bionic for now.

```
E: Unable to locate package python-ldap
E: Package 'python-memcache' has no installation candidate
E: Unable to locate package python-pip
E: Package 'python-twisted' has no installation candidate
```